components/resources/quizzes/langQuiz/LangResults.tsx

```diff
@@ -3,22 +3,70 @@ import SkillifyNavbar from "../shared/SkillifyNavbar";
 
 type LangResultsProps = {
```

Comment:
lets stop using the short form Lang, use the full word Language. It might feel verbose to you and faster to use the short form, but it's more readable to use more descriptive full words